resources/views/event/display_includes/event_block.blade.php

```diff
@@ -24,32 +24,26 @@
             @endif
 
             {{-- Participating --}}
-            @if(Auth::check() && $event->activity?->isParticipating(Auth::user()))
-                @if($event->activity->isOnBackupList(Auth::user()))
+            @if(Auth::check() && $event->activity?->myParticipationCount > 0)
+                @if($event->activity->myBackupParticipationCount > 0)
                     <i class="fas fa-check text-warning fa-fw" aria-hidden="true"
                        data-bs-toggle="tooltip" data-bs-placement="top" title="You are on the backuplist!"></i>
                 @else
                     <i class="fas fa-check text-primary fa-fw" aria-hidden="true"
                        data-bs-toggle="tooltip" data-bs-placement="top" title="You are participating!"></i>
                 @endif
-                @if($event->activity->isHelping(Auth::user()))
+                @if($event->activity->myHelperParticipationCount > 0)
                     <i class="fas fa-life-ring fa-fw text-danger" aria-hidden="true"
                        data-bs-toggle="tooltip" data-bs-placement="top" title="You are helping!"></i>
                 @endif
             @endif
 
             {{-- Ticket --}}
-            @if (Auth::check() && $event->hasBoughtTickets(Auth::user()))
+            @if (Auth::check() && $event->myTicketCount>0)
                 <i class="fas fa-ticket-alt fa-fw text-info" aria-hidden="true"
                    data-bs-toggle="tooltip" data-bs-placement="top" title="You bought a ticket!"></i>
             @endif
 
-            {{-- Helper --}}
```

Comment:
??? why was this removed

Reply:
This should have been removed when we removed the helper reminder emails because we do not use this system anymore